### PR TITLE
Jakarta Annotations 3.0

### PR DESCRIPTION
**Pull request by lukasj**
## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [x] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [x] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [x] Updated release record
           https://projects.eclipse.org/projects/ee4j.ca/releases/3.0
   - [ ] Generated IP Log
   - [x] Email to PMC
   - [x] Start release review by emailing EMO 
- [x] The URL of the OSSRH staging repository for the api, javadoc:
      https://jakarta.oss.sonatype.org/content/repositories/jakartaannotation-1014/
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      http://download.eclipse.org/ee4j/ca/jakartaee11/staged/eftl/jakarta-annotations-tck-3.0.0.zip
- [x] The URL of the compatibility certification request issue:
      https://github.com/jakartaee/common-annotations-api/issues/135
- [x] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

**Comment by lukasj** [comment]
TCK is ready

**Review by ivargrimstad — CHANGES_REQUESTED** [review]
Remove " (Under development)" from the title

Reword the summary to past tense. E.g.

Change 

"The purpose of this release is to fully remove the deprecated @ManagedBean annotation."

to 

"This release fully removes the deprecated @ManagedBean annotation."

**Comment by ivargrimstad** [comment]
@lukasj  Arjan said that GlassFish is planning a 8.0.0-M3 this week that may be used for the CCR for this PR since it incorporates Annotations 3.0

**Comment by lukasj** [comment]
@ivargrimstad comments applied; let me know once M3 is ready

**Comment by lukasj** [comment]
done

File: annotations/3.0/_index.md [hunk 1]
```diff
@@ -23,15 +23,19 @@ The purpose of this release is to fully remove the deprecated @ManagedBean annot
...
-    * [jakarta.annotation:jakarta.annotation-api:jar:3.0.0-M1](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/3.0.0-M1/jar)
+    * [jakarta.annotation:jakarta.annotation-api:jar:3.0.0](https://search.maven.org/artifact/jakarta.annotation/jakarta.annotation-api/3.0.0/jar)
+
+* Compatible Implementations used for [ratification](https://www.eclipse.org/projects/efsp/?version=1.2#efsp-ratification).
+  * TBD
 
 # Compatible Implementations
 
-NA
+TBD
```
Comment (ivargrimstad) [inline annotations/3.0/_index.md:38]: Add GlassFish 8.0.0-M3 here

Comment (lukasj) [reply annotations/3.0/_index.md]: done

**Review by ivargrimstad — APPROVED** [review]

**Comment by ivargrimstad** [comment]
@lukasj  I just talked to @arjantijms . It should be possible to run the TCK with Java 17 and GlassFish 8.0.0-M3 since GlassFish isn't started by the TCK run. Arjan is 95% sure about this.

**Comment by lukasj** [comment]
> @lukasj I just talked to @arjantijms . It should be possible to run the TCK with Java 17 and GlassFish 8.0.0-M3 since GlassFish isn't started by the TCK run. Arjan is 95% sure about this.

the fact it is possible does not mean it is the right way to proceed as it shows reliance on the TCK implementation details, I mean what is the point of certifying a runtime known to NOT support SE 17? But it is the most simple way forward, so done...

**Comment by ivargrimstad** [comment]
* On ballot completion, the specification committee mentor:
 - [ ] adds this final checklist to the main PR.
 - [ ] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
 - [ ] calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
 - [ ] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [ ] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
 - [ ] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
 - [ ] creates an issue in the specification project that includes the following checklist for the specification project team: https://github.com/jakartaee/common-annotations-api/issues/139

**Pull request merged**